README.md

```diff
@@ -109,17 +107,7 @@ python -m lightning run app app.py
 python -m lightning run app app.py --cloud
 ```
 
-If you see the following error while running the app:
```

Comment:
this error is not specific to Lightning but every application which opens a port